regolith/helpers/a_proposalhelper.py

```diff
@@ -128,6 +131,12 @@ def db_updater(self):
                 end_date = date_parser.parse(rc.end_date).date()
                 if begin_date > end_date:
                     raise ValueError("begin_date can not be after end_date")
+                if rc.duration:
+                    expected_duration = 12*relativedelta(end_date, begin_date).years + relativedelta(end_date, begin_date).months
+                    if expected_duration != int(rc.duration):
+                        raise ValueError(f"Entered duration {rc.duration} is inconsistent with"
```

Comment:
I think this is tmi.   Just write  "ERROR: please rerun specifying a duration OR an end-date but not both"

Reply:
done